sdk/core/core-rest-pipeline/src/createPipelineFromOptions.ts

```diff
@@ -88,7 +87,7 @@ export function createPipelineFromOptions(options: InternalPipelineOptions): Pip
     pipeline.addPolicy(decompressResponsePolicy());
   }
 
-  pipeline.addPolicy(formDataPolicy());
+  pipeline.addPolicy(formDataPolicy(), { beforePolicies: [multipartPolicyName] });
```

Comment:
Curious: how come this change was needed?

Reply:
I noticed in some comments that we were relying on ordering to enforce this dependency when the pipeline itself has an explicit feature to let you express when two policies have an ordered relationship, so I added it. Really I should have caught it in the PR that added this line, but ah well.

Reply:
For context, at the time I thought adding this was redundant due to the use of `afterPhase` for `multipartPolicy` but I suppose it's better to be explicit? (side note: using `afterPhase` was sadly necessary since the recorder needs to inject a policy in between the form data policy and the multipart policy. `CommonClientOptions` doesn't expose fine-grained control like `afterPolicy`, `afterPhase`, etc. so this is the best way of doing it that I could think of. Would be very open to alternative approaches :))

Reply:
yeah, the archboard didn't like my fancy policy constraints :(